blockchain.go

```diff
@@ -1351,3 +1420,11 @@ func (b *Blockchain) testAlternativeHashAlgo(sig flowgo.TransactionSignature, ms
 
 	return nil
 }
+
+func (b *Blockchain) SetDebugger(debugger *interpreter.Debugger) {
+	b.debugger = debugger
+}
+
+func (b *Blockchain) EndDebugging() {
```

Comment:
nit: `SetDebugger(nil)`

Reply:
Refactored in 08c266d15d1076e8d8992c7540c9a749225bc5c4

Reply:
I also meant if it makes sense to just reuse the exported SetDebugger function to not actually have additional EndDebugging. Highly subjective so feel free to dismiss or even better educate me 😄